src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/FindAllReferencesHandler.cs

```diff
@@ -163,11 +191,14 @@ async Task DelayAfterLastNotifyAsync(CancellationToken cancellationToken)
 
             if (result == null || result.Length == 0)
             {
+                _logger.LogInformation("Received empty progress notification");
                 return;
             }
 
+            _logger.LogInformation($"Received {result.Length} references, remapping.");
             var remappedResults = await RemapReferenceItemsAsync(result, cancellationToken).ConfigureAwait(false);
 
+            _logger.LogInformation($"Reporting {remappedResults} results.");
```

Comment:
Did you mean `remappedResults.(Count|Length)`? I would expect this to print the whole structure of remappedResults.

Reply:
Yep, good catch, thanks!